### ISSUE TITLE
prov verbs;ofi_rxm: seg faults in rxm_cq_read_write_error() after using fi_writedata() to transfer 264 bytes

### ISSUE THREAD

**Issue by tonyzinger** [ofiwg/libfabric#4734]
After transferring 264 or more bytes via the fi_writedata API and using the verbs:ofi_rxm provider, a segmentation fault occurs in rxm_cq_read_write_error() while trying to process the CQ event for this transaction.  If the transfer size is 256 bytes or less this problem does not occur.

This problem does not occur with the sockets provider.

The top of the stack from the core dump shows:
```
#0  0x00007fc83aa1e9f8 in rxm_cq_read_write_error (rxm_ep=0x791580) at prov/rxm/src/rxm_cq.c:1150
#1  0x00007fc83aa1f2f7 in rxm_ep_do_progress (util_ep=0x791580) at prov/rxm/src/rxm_cq.c:1297
#2  0x00007fc83aa1f3eb in rxm_ep_progress (util_ep=0x791580) at prov/rxm/src/rxm_cq.c:1314
#3  0x00007fc83a9758af in ofi_cq_progress (cq=0x792cd0) at prov/util/src/util_cq.c:597
#4  0x00007fc83a974b12 in ofi_cq_readfrom (cq_fid=0x792cd0, buf=0x7ffe881034d0, count=1, src_addr=0x0) at prov/util/src/util_cq.c:245
#5  0x00007fc83a974e02 in ofi_cq_read (cq_fid=0x792cd0, buf=0x7ffe881034d0, count=1) at prov/util/src/util_cq.c:312
#6  0x00007fc83b709fe4 in fi_cq_read (count=1, buf=0x7ffe881034d0, cq=0x792cd0) at /scratch/ajz/install/include/rdma/fi_eq.h:385
```

I am attaching a file that contains debug output from libfabric and test debug output.
[debug_output_ofiwg.txt](https://github.com/ofiwg/libfabric/files/2741540/debug_output_ofiwg.txt)

**Comment by swelch** [comment]
@dmitrygladkov - This issue was fixed with PR #4739  and should be able to be closed. Thanks.

**Comment by dmitrygladkov** [comment]
@swelch Thank you so much!
